### ISSUE TITLE
Crash on game list loading

### ISSUE THREAD

**Issue by chernyaknikita** [flavioislima/HeroicGamesLauncher#7]
I'd logged in and Heroic had started to load my library but after a few seconds it got closed. If I run Heroic again the same thing happens (also it asks me again if I want to log in though no filling needed after my consent).
![issie0635](https://user-images.githubusercontent.com/43153062/103445843-0d768100-4c8a-11eb-9c1b-dbc6c5a926ab.png)
![issie0672](https://user-images.githubusercontent.com/43153062/103445854-149d8f00-4c8a-11eb-806d-285dbed8e306.png)

**Comment by chernyaknikita** [comment]
upd
Got the same thing with local pacman package installation except that now Heroic restarts automaticly after the crush.

**Comment by flavioislima** [comment]
Yeah, I have a fix for that, temporary because I'll change the way the login is done.
I'll release this fix soon.

**Comment by chernyaknikita** [comment]
It didn't work with today's 0.3.2 except there were more info but it doesn't work with 0.4.0 neither. When I've started Heroic there was just a blank window so I deleted Heroic and Legendary configs. After that Heroic showed me login page so I logged in (successfully) but library still wasn't there. I restarted Heroic but got blank screen again.

**Comment by chernyaknikita** [comment]
Also I've followed the advice from readme 

> If you think is easier, before launching the app, run legendary auth on a terminal and after login run legendary list-games. Then the app will identify the configuration and will open your library.

But there's no Legendary in PATH. So I installed it from AUR, recreated config and the blank screen apeared in this case either.

**Comment by flavioislima** [comment]
ok, I think we were talking on Reddit.

So, it's not that intuitive and I'll improve this soon, but once you log in, you can click on your username on the top right and it will get you to the library page. There you should be able to see your games.

But if you close the application, and open it again, on the Library there are no games?

Can you list the files under `.config/legendary/metadata` ? In this folder should be all the games that you have, one file per game. 
Also, since you have `legendary` installed now, you can also try to put on the terminal: `legendary list-games`  and see if there is any output.

**Comment by flavioislima** [comment]
@chernyaknikita I think I might have found the issue. Can you try now with this AppImage:
https://drive.google.com/file/d/1Ecdai5UY2fEK-oa1QhtfMR-k0QQWxxzE/view?usp=sharing

if it works I'll reçease the fix officially.

I think this is because some Game/DLC does not have an image, so it is breaking the logic when listing the games. So I added a fallback image to deal with that.

**Comment by chernyaknikita** [comment]
> ok, I think we were talking on Reddit.
> 
> So, it's not that intuitive and I'll improve this soon, but once you log in, you can click on your username on the top right and it will get you to the library page. There you should be able to see your games.
> 
> But if you close the application, and open it again, on the Library there are no games?
> 
> Can you list the files under `.config/legendary/metadata` ? In this folder should be all the games that you have, one file per game.
> Also, since you have `legendary` installed now, you can also try to put on the terminal: `legendary list-games` and see if there is any output.

There was an output and I had all games in config folder. Anyway, the new version has fixed it, thx!
P.S. We weren't talking on Reddit :D

**Comment by flavioislima** [comment]
Ah, nice!

But one question, have you tried login in using Heroic again with this version I sent you?
If possible, I appreciate if you could try that.
You don't need to remove the legendary folder, just rename it and then try to login and wait until the game load the library and click on your username on the top.

**Comment by chernyaknikita** [comment]
Yes, it's fine now. If you click on the nickname before game list loads this page appears. If you click "update game list" nothing visually happens except the button outline but after a while the list loads (I believe, even automaticly without me clicking that button). Though if you click more than one time on that button you can't understand if it was actially clicked because the button outline isn't changing in any way.
![image](https://user-images.githubusercontent.com/43153062/103492370-80732980-4e3b-11eb-9b8f-767c701d8082.png)

**Comment by StoneyFish** [comment]
I was having the same issue getting a blank black screen with 4.0 Deb and AppImage. For me the test 4.1 AppImage is now listing all my games. Thanks.

**Comment by whyareugoodman** [comment]
when i sign in to epic it saying loading in... i was waiting for an hour-_-
